autoload/lsp/utils/text_edit.vim

```diff
@@ -71,6 +72,9 @@ function! lsp#utils#text_edit#apply_text_edits(uri, text_edits) abort
             set virtualedit=onemore
 
             silent execute l:cmd
+            if a:options['show_edits']
```

Comment:
Maybe you can use something like `get(a:options, 'show_edits', 0)` so that it is default `0`. This way, you don't need to add the option explicitly in other locations.

Reply:
I've not done a lot of vimscript so wasn't sure what the convention was, so I went with personal preference.

Reply:
Do you mean passing `{}` instead of `{'show_edits': 0}`?

Reply:
Ah right, you still need the `{}`. You could do something like this:
```vim
function! lsp#utils#text_edit#apply_text_edits(uri, text_edits, options) abort
    let l:options = get(a:, 1, {})
    let l:show_edits = get(l:options, 'show_edits', 0)

    " ...
endfunction
```
but I'm fine with passing the `0` explicitly too.

Reply:
See `:h a:` if you're unfamiliar with the `...` or `a:` and would like to learn more :smile: